lib/presentation/config/pages_of_presentation.dart

```diff
@@ -9,40 +10,67 @@ import 'package:flutter_show/slides/06_outro/view/outo_slide.dart';
 enum PagesOfPresentation {
   titleSlide(
     slide: TitleSlide(),
-    items: 1,
+    animationSteps: 1,
     title: 'Title',
   ),
   agendaSlide(
     slide: AgendaSlide(),
-    items: 1,
+    animationSteps: 1,
     title: 'Agenda',
   ),
   motivation(
     slide: MotivationSlide(),
-    items: 4,
+    animationSteps: 4,
   ),
   downsides(
     slide: DownsidesSlide(),
-    items: 6,
+    animationSteps: 6,
     title: 'Downsidessssss',
   ),
   benefits(
     slide: BenefitsSlide(),
-    items: 6,
+    animationSteps: 6,
   ),
   outro(
     slide: OutroSlide(),
-    items: 1,
+    animationSteps: 1,
     title: 'Fin',
   );
 
   const PagesOfPresentation({
     required this.slide,
-    required this.items,
+    required this.animationSteps,
```

Comment:
I am thinking, instead of mentioning animation steps for every slide. We can make the default as `1` step and then users can add custom steps when needed.

Documentation changes also are needed after this change.
```suggestion
    this.animationSteps = 1,
```